letsencrypt-win-simple/ScheduledRenewal.cs

```diff
@@ -12,6 +12,7 @@ public class ScheduledRenewal
         public string KeepExisting { get; set; }
         public string Script { get; set; }
         public string ScriptParameters { get; set; }
+        public bool Warmup { get; set; }
```

Comment:
The serialization/deserialization is working for me as a boolean, it gets written as:
`"Warmup":true`